### DISCUSSION TITLE
discussion test

### DISCUSSION THREAD

**Discussion by ExperimentsInHonesty** [hackforla/website#509] [General] [answered]
### Overview
REPLACE THIS TEXT -Text here that clearly states the purpose of this issue in 2 sentences or less.

### Action Items
REPLACE THIS TEXT -If this is the beginning of the task this is most likely something to be researched and documented.

REPLACE THIS TEXT -If the issue has already been researched, and the course of action is clear, this will describe the steps.  However, if the steps can be divided into tasks for more than one person, we recommend dividing it up into separate issues, or assigning it as a pair programming task.

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

**Comment by cnk** [comment]
What should we do to test using this GitHub feature?

**Comment by akibrhast** [comment, marked as answer]
testing commenting on a discussion
